pkg/consumer/consumer.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2021 VMware, Inc.
```

Comment:
Let's keep both producer and consumer in pkg/kafka?

Reply:
Sure. Updated